gdx/src/com/badlogic/gdx/utils/Queue.java

```diff
@@ -368,6 +390,8 @@ public String toString (String separator) {
 		return sb.toString();
 	}
 
+	/** Method to transform the queue into a hashCode.
+	 * @return the hashCode of the queue */
```

Comment:
no need to javadoc overriden methods if there is no specific implementation information.
add @Override annotation instead (same for equals method)